test/GogeDao.t.sol

```diff
@@ -130,13 +162,7 @@ contract DaoTest is Utility, Test {
         assertEq(gogeToken.balanceOf(address(joe)), 1_000_000_000 ether);
 
         // Approve tokens for vote.
-        assert(
-            joe.try_approveToken(
-                address(gogeToken),
-                address(gogeDao),
-                1_000_000_000 ether
-            )
-        );
+        assert(joe.try_approveToken(address(gogeToken), address(gogeDao), 1_000_000_000 ether));
 
         // Verify Joe cannot make more votes than the balance in his wallet.
         assert(!joe.try_addVote(address(gogeDao), 1, 1_000_000_000 ether + 1));
```

Comment:
I would like to see the use of `vm.expectRevert()` here or in a separate test(s) to make sure the specific revert condition you're testing against triggers as well as simulate how each revert condition can be triggered.

Reply:
Will come back to this